specifyweb/frontend/js_src/lib/components/Formatters/Fields.tsx

```diff
@@ -117,7 +117,6 @@ export function Fields({
           >
             {resourcesText.addField()}
           </Button.Secondary>
-          <span className="-ml-2 flex-1" />
```

Comment:
this was needed to have the customize field formatters check box on the bottom right of component. Why moving it next to the add field button?

Reply:
![Screenshot 2024-03-11 at 7 42 26 AM](https://github.com/specify/specify7/assets/108160931/1119823c-752a-4c34-a2f1-0cac26633420)
![Screenshot 2024-03-11 at 7 43 06 AM](https://github.com/specify/specify7/assets/108160931/995f2fd1-89ba-48b5-a03a-02ff899dcdeb)

Reply:
Bronwyn prefers it the way it was as well, so I will return it.